Containerfile

```diff
@@ -45,7 +47,7 @@ FROM ${BUILD_TYPE}_image AS final_image
 ENTRYPOINT ["model_signing"]
 CMD ["--help"]
 
-ARG APP_VERSION="1.0.1"
+ARG APP_VERSION="1.1.1"
```

Comment:
@mihaimaruseac we should have a better process to make sure we dont need to do this or forget to do it